newIDE/app/src/LoginProvider/LocalLoginProvider.js

```diff
@@ -61,11 +61,6 @@ class LocalLoginProvider implements LoginProvider, FirebaseBasedLoginProvider {
       if (signal) {
         signal.addEventListener('abort', () => {
           terminateWebSocket();
-          reject(
```

Comment:
this seemed duplicated in this file and the browser one.
This was always causing an error in the console because the signal always aborts (we trigger it when we close the dialog)
I don't think this deserves to raise an error as it's the expected path?

Reply:
Expected or not, the promise must resolve and here it will never resolve/reject?

Reply:
Which means that this will never call then or catch:
<img width="730" alt="image" src="https://github.com/user-attachments/assets/79ad6d88-46e0-4240-9915-e41ea92ca1ea">

Which means the callers of loginWithProvider will never finish:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/5888bc21-bf34-4d91-91fb-c4dc0e0ea692">

You might want to resolve instead of rejecting, or reject with an error that is an object like {cancelled: true, error: null} or `{cancelled: false, error: ...}`.
but the promise should still finish at some point?

Reply:
true, I removed the reject too quickly.

Reply:
turns out this was really needed and this error was already handled properly, I just needed to avoid logging it as an error.